cli/popper/commands/cmd_run.py

```diff
@@ -120,124 +112,148 @@
     default=popper.scm.get_git_root_folder()
 )
 @pass_context
-def cli(ctx, action, wfile, skip_clone, skip_pull, skip, reuse,
-        recursive, quiet, debug, dry_run, parallel, log_file,
-        with_dependencies, on_failure, runtime, workspace):
-    """Execute github actions workflows.
+def cli(ctx, **kwargs):
+    """Executes one or more pipelines and reports on their status.
```

Comment:
we can explain here a bit more of what the command does, making references to what the flags do, at a high-level